### DISCUSSION TITLE
Left turn issue MP

### DISCUSSION THREAD

**Discussion by tgurok** [Mogli12/KeyboardSteer#777] [Q&A]
For some reason, when I do a left turn with Alt-A it does a lane change, not a 180 to the left, turning right with Alt-d works correctly.  It's a MP hosted server.

The guy I play with has no issues with all the same settings.

Any ideas?  Thank you.

**Comment by Mogli12** [comment]
Sometimes it happens that Alt+A or Alt+D turns the steering wheel, which disables guidance steering. You can enabled it again with Alt+S.

**Reply by tgurok** [reply]
This isn't it, it just moves the lane over, and keeps the same direction instead of doing the 180.  I have to disengage, manually turn and re-enable.  The lane still stays blue, shows the arrows keeping the same direction.  It's very odd, it didn't do this until about two days ago, and it's not consistent. It now happens more than not.

**Comment by Mogli12** [comment]
Did you drive this vehicle all the time? Can you make a video? Are you using the latest build (44)?

**Reply by tgurok** [reply]
I was trying to make a video and it stopped doing it, but I did also change A and D not to steer?  Who knows if it happens again I'll make a video.  I think we are on 44, yes.
